config/application.rb

```diff
@@ -89,6 +89,13 @@ class Application < Rails::Application
     # Determine if GreenLight should enable email verification
     config.enable_email_verification = (ENV['ALLOW_MAIL_NOTIFICATIONS'] == "true")
 
+    # Determine if GreenLight should require a certain mail-domain
+    config.require_email_domain = if ENV["REQUIRE_MAIL_DOMAIN"].present?
```

Comment:
This can probably be set to `config.require_email_domain = ENV["REQUIRE_MAIL_DOMAIN"].split(,)` (see below)